### ISSUE TITLE
[database > UsersCollection] Add `glossSuggestion: "On"` to every user

### ISSUE THREAD

**Issue by imnasnainaec** [sillsdev/TheCombine#3315]
Follow-up to #3229

- [ ] Create and test mongo command locally
   - Add to https://docs.google.com/document/d/1L9WjBZqBNVfxk5KiIQo5Fzjsetv4JuwGU0wuYKDRtWo
- [ ] On QA after #3229 is merged into `master`
- [ ] On live server after next release

**Comment by imnasnainaec** [comment]
Actually, the missing field is filled in with the desired default value as Users are logged in, so this is an unnecessary database change.
